src/pages/home/atoms/Artist.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Composition } from 'atomic-layout'
+import styled from 'styled-components'
+import Text from '../../../atoms/Text'
+import { Link } from 'react-router-dom'
+
+const SryledImageContainer = styled.div`
+  width: 100%;
+  border-radius: 50%;
+  overflow: hidden;
+`
+
+const areasMobile = `
+	img
+	text
+`
+
+const Artist = ({ name, artistId, src }) => {
+  return (
+    <Composition areas={areasMobile}>
+      {Areas => (
+        <Link to={artistId}>
```

Comment:
This will create a link like `/ASklhoaho83q`. If you want for this link to go to the artist page, it needs to be the same as you have in the routes:

https://github.com/dandelionadia/tones/blob/dfaebffb7edccb9d42d970589c828019ab63cf64/src/App.js#L40

So `/artist/ASklhoaho83q`.